src/coreclr/src/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -2547,6 +2547,13 @@ private void MethodCompileComplete(CORINFO_METHOD_STRUCT_* methHnd)
             return null;
         }
 
+        private bool getTailCallHelpers(ref CORINFO_RESOLVED_TOKEN callToken, CORINFO_SIG_INFO* sig, CORINFO_GET_TAILCALL_HELPERS_FLAGS flags, ref CORINFO_TAILCALL_HELPERS pResult)
+        {
+            // Slow tailcalls are not supported yet
+            // https://github.com/dotnet/corert/issues/1683
```

Comment:
Should we migrate this issue over to dotnet/runtime?

Reply:
Opened https://github.com/dotnet/runtime/issues/35423